server/api/member_test.go

```diff
@@ -128,7 +152,9 @@ func (s *testMemberAPISuite) TestMemberList(c *C) {
 		defer clean()
 
 		parts := []string{cfgs[rand.Intn(len(cfgs))].ClientUrls, apiPrefix, "/api/v1/members"}
-		resp, err := s.hc.Get(strings.Join(parts, ""))
+		addr, err := unixAddrToHTTPAddr(strings.Join(parts, ""))
```

Comment:
I think it's better to modify `ClientUrls` in `mustNewCluster`, instead of converting it everywhere.